### PR TITLE
UbiquitiInc.UI version 0.50.1.2

### PR DESCRIPTION
**Pull request by walderston**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/57788)

### CODE REVIEW

File: manifests/u/UbiquitiInc/UI/0.50.1.2/UbiquitiInc.UI.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: UbiquitiInc.UI
+PackageVersion: 0.50.1.2
+PackageLocale: en-US
+Publisher: Ubiquiti Inc
+PackageName: UI
+License: Copyright (c) Ubiquiti Inc. All rights reserved.
```
Comment (Trenly) [inline manifests/u/UbiquitiInc/UI/0.50.1.2/UbiquitiInc.UI.locale.en-US.yaml:9]: A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by walderston** [comment]
The domain ubnt.com forwards to ui.com. exception to this is the download area of Ubiquity which is still hosted on ubnt.com

**Comment by lechacon** [comment]
@walderston I'm not able to find the download link in the publisher page to confirm the URI. Could you provide a link to the downloads page where you got this URI from? I did confirm that ubnt.com forwards to ui.com, but I'd want to see something pointing from ui.com to ubnt.com to be completely sure.

**Comment by walderston** [comment]
> @walderston I'm not able to find the download link in the publisher page to confirm the URI. Could you provide a link to the downloads page where you got this URI from? I did confirm that ubnt.com forwards to ui.com, but I'd want to see something pointing from ui.com to ubnt.com to be completely sure.

You have to be signed into a UI account to be shown the download links for their new application 'UI Desktop'.

Doing a registration lookup on ubnt.com shows it is owned by Ubiquiti Networks the same as UI.com - https://lookup.icann.org/lookup?q=ubnt.com&t=a (registration last updated in 2018 and valid to 2026).

Also looking at an archive from 2018 shows the main website on ubnt.com. In 2018, they started to move things over to UI.com however have not moved over all their downloads. - such as https://web.archive.org/web/20180223141805/https://www.ubnt.com/

Also looking at the community support site shows many threads about both the move from ubnt.com to ui.com such as [XX ](https://community.ui.com/questions/UBNT-com-Changed-to-UI-com/491f0b28-4d2e-4dbd-9e83-72a10f1ddf50#answer/5c376264-f526-4c33-af44-d52ff591df33) and general support issues around using the ubnt.com page (https://community.ui.com/questions/unifi-crededentials-not-working-at-ui-com-but-ubnt-com-seems-OK/0c4eac0c-10f9-4147-8d3e-a18d6e3b6b92)

If this isn't good enough, I'll close this PR.

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**